src/Hilke.EFloatCalculator/EFloatAlgebraicNumberCalculator.cs

```diff
@@ -0,0 +1,131 @@
+using System;
```

Comment:
The folder is wrong, it should be called `Hilke.KineticConvolution.EFloatCalculator`